File: ComputersApp.Api/ClientApp/src/app/core/models/computer.ts
import { OsType } from './OsType.enum';
import { ICpu } from './cpu';
export interface IComputer {
    id?: number;
    name: string;
    cpu?: ICpu;
    cpuId?: number;
    osType?: OsType;
    ramAmount: number;
    ssdAmount: number;
}

File: ComputersApp.CosmoDb/Models/Computer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ComputersApp.CosmoDb.Models
{
    public class Computer
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public Cpu Cpu { get; set; }
        public int? CpuId { get; set; }
        public OsType OsType { get; set; }
        public double RamAmount { get; set; }
        public double SsdAmount { get; set; }
    }
}
